templates/package.json.template

```diff
@@ -34,9 +34,10 @@
     "dependencies": {
       "arguejs": "^0.2.3",
       "lodash": "^4.15.0",
+      "long": "^3.2.0",
```

Comment:
Where are you using the long module? If you're not using it, please remove it, and if you're only using it for tests, please move it to `devDependencies`.